packaging/ubuntu-14.04/rules

```diff
@@ -103,6 +103,8 @@ override_dh_auto_build:
 	cd cmd && ( autoreconf -i -f )
 	cd cmd && ( ./configure --prefix=/usr --libexecdir=/usr/lib/snapd $(VENDOR_ARGS))
 	$(MAKE) -C cmd all
+	# Build any available upstart configurations
+	$(MAKE) -C data/upstart all
```

Comment:
Isn't that already a variable set in the Makefile which can be optionally overridden by the caller?